### ISSUE TITLE
Create the padding-top class in base component

### ISSUE THREAD

**Issue by fabiorogeriosj** [MobileUI/components#90]
.padding-top

**Comment by daniloff200** [comment]
what value of padding do we need?

**Comment by fabiorogeriosj** [comment]
@daniloff200 10px, equal margin-top:

https://github.com/MobileUI/components/blob/master/base/mobileui.css

```
/*Add a margin on the top.*/
.margin-top {
  margin-top: 10px;
}
```

**Comment by galloj** [comment]
@fabiorogeriosj This issue can be closed #94
